examples/codes/run_plasmod_example.py

```diff
@@ -215,6 +215,15 @@ def plot_profile(solver, profile, var_unit):
 plot_profile(solver, plasmod.Profiles.Te, "keV")
 print_outputs(solver)
 
+# %%[markdown]
+# ### Plotting the results
+# There is a default set of output profiles that can be plotted easily:
+
+# %%
+
+plasmod.plot_default_profiles(solver)
```

Comment:
Would it be more flexible to instead plot from a `PlasmodOutputs` object. I don't think the plotting function needs anything else from the solver.

So this would instead be

```python
plasmod.plot_default_profiles(solver.plasmod_outputs())
```

This way you get a little more flexibilty, and could, in theory, plot the profiles straight from the plasmod output files (using `PlasmodOutputs.from_files`), without running the files through the solver.